tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -609,6 +627,18 @@ platform_tests/test_reboot.py:
     conditions:
       - "hwsku in ['Celestica-E1031-T48S4']"
 
+platform_tests/test_reboot.py::test_power_off_reboot:
+  skip:
+    reason: "Skip power off reboot test for Wistron"
+    conditions:
+      - "'sw_to3200k' in hwsku"
+
+platform_tests/test_reboot.py::test_watchdog_reboot:
+  skip:
+    reason: "Skip watchdog reboot test for Wistron"
+    conditions:
+      - "'sw_to3200k' in hwsku"
```

Comment:
Similar issue here.

Reply:
modified this according the suggestion you provided.

Thanks.